Applications/Utils/MeshGeoTools/ConstructMeshesFromGeometry.cpp

```diff
@@ -0,0 +1,72 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include <tclap/CmdLine.h>
+
+#include "Applications/ApplicationsLib/LogogSetup.h"
+#include "GeoLib/GEOObjects.h"
+#include "GeoLib/IO/XmlIO/Boost/BoostXmlGmlInterface.h"
+#include "MeshGeoToolsLib/ConstructMeshesFromGeometries.h"
+#include "MeshGeoToolsLib/SearchLength.h"
+#include "MeshLib/IO/readMeshFromFile.h"
+#include "MeshLib/IO/writeMeshToFile.h"
+#include "MeshLib/Mesh.h"
+
+std::unique_ptr<GeoLib::GEOObjects> readGeometry(std::string const& filename)
+{
+    auto geo_objects = std::make_unique<GeoLib::GEOObjects>();
+    GeoLib::IO::BoostXmlGmlInterface gml_reader(*geo_objects);
+
+    DBUG("Reading geometry file \'%s\'.", filename.c_str());
+    gml_reader.readFile(filename);
+    return geo_objects;
+}
+
+int main(int argc, char* argv[])
+{
+    ApplicationsLib::LogogSetup logo_setup;
+
+    TCLAP::CmdLine cmd(
+        "Converts a geometry defined on a given mesh to distinct meshes.",
```

Comment:
The docu on the website is missing. Then it would be nice to add a link to the docu in the short tool description.